Update react example

bwasty opened the pull request:
I tried out the react example and made some fixes/improvements:
- fix path for @parity/light.js (package.json)
- update dependencies
  - skipped (breaking changes for 2.x): react-scripts, react-app-rewired (-> breaks decorators*)
- remove hint about window.parity (not working)
- make font sans-serif

\* About the decorators: I would suggest removing them or converting to TypeScript. I think such a simple example should use a 'plain' create-react-app setup, without things like `react-app-rewired`/config overrides.

amaurymartiny commented:
Thanks a lot! Yes, the example might have been left out a bit, thanks for having a look.

>  I think such a simple example should use a 'plain' create-react-app setup

You're right, I wanted to show "look how cool this lib is with decorators", but for such a simple example, it might be better to drop them. Would you mind doing this in this PR? At the same time remove react-app-rewired and update to CRA2 (plain JS). Cheers!

bwasty commented:
Done!
I actually re-created the example into `example-react` (with `yarn create react-app`), then copied over the relevant code and deleted the old version. I think `-react` makes sense since the Angular example (#60) could then go into `example-angular`.
I also switched from yarn to npm and committed the `package-lock.json` - I had lots of problems with yarn (super slow - 10min for install if it works, mostly it failed or hung...).
